DOC: Explain how to opt-out of boot image update in 4.18 (#93167)
openshift/openshift-docs: modules/mco-update-boot-images-disable.adoc

[id="mco-update-boot-images-disable_{context}"]
= Disabling updated boot images

> To disable the updated boot image feature, edit the `MachineConfiguration` object to remove the `managedBootImages` parameters.

Reviewer: ```suggestion
To disable the updated boot image feature, edit the `MachineConfiguration` object to have an empty list for the `machineMangers` field.
```

> To disable the updated boot image feature, edit the `MachineConfiguration` object to have an empty list for the `machineMangers` field.

Reviewer 2: machineMangers -> MachineManagers

> To disable the updated boot image feature, edit the `MachineConfiguration` object to have an empty array for the `machineMangers` field.

Reviewer 3: This is just a suggestion to make the text clearer: change from the descriptive to the active. 

```suggestion
To disable the updated boot image feature, edit the `MachineConfiguration` object so that  the `machineMangers` field is an empty array .
```

Reviewer 4: Not against this suggestion unless there's a specific reason to keep the wording as-is, just note that the suggested text has an extra space in between "that" and "the `machineMangers`"

> To disable the updated boot image feature, edit the `MachineConfiguration` object to have an empty array for the `machineMangers` field.

Reviewer 4: Also is this a typo?
```suggestion
To disable the updated boot image feature, edit the `MachineConfiguration` object to have an empty array for the `machineManagers` field.
```

spec:
# ...
  managedBootImages: <1>
    machineManagers: []
----

> <1> Remove the parameters under `machineManagers` and add the `[]` characters to disable updated boot images.

Reviewer 3: This is just a suggestion to make the text clearer. 

```suggestion
<1> Remove the parameters  that are listed under `machineManagers` and add the `[]` characters to disable the updated boot images.
```
